framework/chi_runtime.cc

```diff
@@ -213,7 +212,7 @@ void chi::Finalize()
   meshhandler_stack.clear();
 
   surface_mesh_stack.clear();
-  logicvolume_stack.clear();
```

Comment:
`object_stack` is cleared below. This line should be able to be omitted.

Reply:
So here it shows up as red for me. Meaning it was indeed removed. Does it show different for you?

Reply:
It shows as red for me here right now, but before it showed a green line on the other side that said `object_stack.clear();`. Weird. All good.